compute_endpoint/globus_compute_endpoint/endpoint/config/model.py

```diff
@@ -184,3 +178,10 @@ def dict(self, *args, **kwargs):
         engine = ret.pop("engine", None)
         ret["executors"] = [engine] if engine else None
         return ret
+
+
+class TemplateConfigModel(BaseConfigModel):
```

Comment:
I think we should match the name of the related `EndpointConfig` model:
```suggestion
class EndpointConfigModel(BaseConfigModel):
```

Reply:
Och; I missed this rename after our conversation.  I'll ignore this because there are a couple of places to rename.  Will push shortly.  Thanks.